pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -50,31 +66,36 @@ class Ametek7270(Instrument):
     ]
 
     sensitivity = Instrument.control(  # NOTE: only for IMODE = 1.
-        "SEN.", "SEN %d",
+        "SEN", "SEN %d",
         """ A floating point property that controls the sensitivity
         range in Volts, which can take discrete values from 2 nV to
         1 V. This property can be set. """,
         validator=truncated_discrete_set,
         values=SENSITIVITIES,
-        map_values=True
+        map_values=True,
+        check_set_errors=True,
+        dynamic=True

```

Comment:
```suggestion
        dynamic=True,

```
It's good practice to finish even the last line with a comma. That way, you do not have to touch lines (see above `map_values=True`, while adding/removing unrelated lines.